### ISSUE TITLE
Thumbnails disappear with more than 20 views

### ISSUE THREAD

**Issue by eugeneniemand** [structurizr/lite#2]
With 20 or less views the navigation pane renders thumbnails like this
![image](https://user-images.githubusercontent.com/6813309/189064625-4274a930-2519-43a9-961f-1ff6f0802b29.png)
However add one more view and they disappear and looks like this
![image](https://user-images.githubusercontent.com/6813309/189064865-98e69d62-8970-4354-8ebb-22435a6c4028.png)
Is this limitation by design or a bug?

**Comment by simonbrowndotje** [comment]
It's a limitation by design ... the thumbnails are served up dynamically, and there's a performance impact when too many are shown.

**Comment by eugeneniemand** [comment]
Ah ok, it would be nice to have this a configuration option to allow the consumer to decide if they are happy with the performance impact, alternatively do some lazy loading as you scroll to only load the thumbnails when coming into view. 20 thumbnails on my machine may be very different than 20 on your machine; I assume the complexity of the view may also have an impact. At a minimum I would like this to be in the docs as it took me some time to realise this was the issue. I reverted all my changes, then pulled the latest image, then tried the main branch and only after speaking to a colleague figured out there is an arbitrary limit.

**Comment by simonbrowndotje** [comment]
The limit has now been removed.
